packages/bff/.env

```diff
@@ -11,3 +11,25 @@ CATALOG_PROCESS_URL="http://localhost:3000"
 ATTRIBUTE_REGISTRY_PROCESS_URL="http://localhost:3000"
 PURPOSE_PROCESS_URL="http://localhost:3000"
 
+TENANT_ALLOWED_ORIGINS="IPA"
+
+ALLOW_LIST_CONTAINER="interop-local-bucket"
+ALLOW_LIST_PATH="allow-list"
+ALLOW_LIST_FILE="allow-list"
+
+INTERNAL_JWT_KID=17c1177f-d7dc-4181-9f54-6fd416bf229b
+INTERNAL_JWT_SUBJECT="dev-refactor.interop-eservice-descriptors-archiver"
+INTERNAL_JWT_ISSUER="dev-refactor.interop.pagopa.it"
+INTERNAL_JWT_AUDIENCE="refactor.dev.interop.pagopa.it/internal"
+INTERNAL_JWT_SECONDS_DURATION=60
+
+GENERATED_JWT_KID=17c1177f-d7dc-4181-9f54-6fd416bf229b
+GENERATED_JWT_SUBJECT="dev-refactor.interop-eservice-descriptors-archiver"
+GENERATED_JWT_ISSUER="dev-refactor.interop.pagopa.it"
+GENERATED_JWT_AUDIENCE="refactor.dev.interop.pagopa.it/internal"
+GENERATED_JWT_SECONDS_DURATION=60
+
+S3_CUSTOM_SERVER=true
+S3_SERVER_HOST=http://localhost
+S3_SERVER_PORT=9000
```

Comment:
@micdes-pagopa these variables can be interesting regarding K8s config.

Reply:
These should be used only in local testing, the AWS SDK doesn't need them to use the real S3